src/omega_edit/client.ts

```diff
@@ -113,17 +121,19 @@ async function serverStart(ctx, version) {
     ++animationFrame
   }, animationInterval)
 
+  // if (oemgaEditServerPort === 0) throw 'Server port was never set'
```

Comment:
remove this

Reply:
Had to step out for a house showing, should be home to make this change soon.